### PR TITLE
feat: display subscriptions table on poke

### PR DESCRIPTION
**Pull request by fontanierh**
## Description

https://github.com/dust-tt/tasks/issues/572

Screenshot 2024-03-28 at 19 45 21

displays a list of all subscriptions on Poke. For now, the whole JSON of the active subscription is kept. We'll likely want to get rid of it or at least collapse it by default

### CODE REVIEW

**Comment by spolu** [comment]
wrong screenshot :)

**Review by spolu — APPROVED** [review]

**Review by PopDaph — APPROVED** [review]
🙏🏻

File: front/pages/poke/[wId]/index.tsx [hunk 1]
```diff
@@ -302,7 +332,7 @@ const WorkspacePage = ({
               </div>
 
               <h2 className="text-md mb-4 mt-8 font-bold">Plan:</h2>
-              <JsonViewer value={subscription} rootName={false} />
+              <JsonViewer value={activeSubscription} rootName={false} />
```
[the remark this answers is not in the document]
Comment (fontanierh) [reply front/pages/poke/[wId]/index.tsx:335]: This still has more info, no strong opinion we could indeed remove

[the remark this answers is not in the document]
Comment (fontanierh) [reply front/pages/poke/[wId]/index.tsx:334]: Will remove in subsequent PR once I am sure no one is missing anything

**Pull request merged**